sklearn/base.py

```diff
@@ -558,8 +558,7 @@ class RegressorMixin:
     _estimator_type = "regressor"
 
     def score(self, X, y, sample_weight=None):
-        """Return the coefficient of determination :math:`R^2` of the
-        prediction.
+        """Coefficient of determination :math:`R^2` of the prediction.
```

Comment:
```suggestion
        """Return the coefficient of determination :math:`R^2` of the prediction.
```
This was fixed in another PR and shouldn't be too long. It's still important to have the verb.